content/docs/introduction/requirements.md

```diff
@@ -146,7 +146,16 @@ The table below shows which version of mSupply you will require when running Ope
 | 1.1.00 - 1.1.16  | 7.04.01+              |
 | 1.2.00+          | 7.05.05+              |
 | 1.4.00+          | 7.09.00+              |
+| 2.0.00+          | 7.14.00+              |
 
 If you attempt to connect to an incompatible server you'll get an error message like this:
 
 ![server version mismatch!](/docs/introduction/images/version_mismatch.png)
+
+## Open mSupply Central Server
```

Comment:
To be in line with other usages, and the fact that `central` and `server` aren't proper nouns in themselves, and nor is `central server` in this case
```suggestion
## Open mSupply central server
```